### ISSUE TITLE
Synthetic Fuel Energy Variables

### ISSUE THREAD

**Issue by fschreyer** [IAMconsortium/common-definitions#84]
Dear all,

we have had internally some discussions about the naming convention of synthetic liquids and gases, which we would like to address on the IAMC-level again. The issue is related to point 2.) raised here #47 .

Currently, we have the following distinctions:
```
- Secondary Energy|Liquids|Biomass:
    description: Production of liquid fuels from biomass
    unit: EJ/yr
- Secondary Energy|Liquids|{Primary Fossil Fuel}:
    description: Production of fossil liquid fuels from {Primary Fossil Fuel}
    unit: EJ/yr
- Secondary Energy|Liquids|Electricity:
    description: Production of liquid fuels from electricity (e-fuels)
    unit: EJ/yr
- Secondary Energy|Liquids|Other:
    description: Production of liquids fuels from sources that do not fit any other category
    unit: EJ/yr
```

``Secondary Energy|Liquids|Electricity`` only includes synthetic fuels where the hydrogen was made via electrolysis from electricity. However, this comes with some problems and ambiguities considering the way synthetic fuels may be implemented in models and how their production may work in the real world. I'd rather opt for something like ``Secondary Energy|Liquids|Hydrogen`` or ``Secondary Energy|Liquids|Hydrogen w/ CCU`` instead. Here are the reasons:

1.) Synthetic fuels are not directly produced from electricity but there is hydrogen as an intermediate product, which is also an energy carrier in the IAMC convention and shows up as ``Secondary Energy|Hydrogen``. It is therefore weird to simply skip this energy carrier, given that the hydrogen needed for synthetic fuels should also be accounted under the SE hydrogen variable (in my view). Also, it is supposedly how some models implement it, i.e. synthetic fuels as a product of hydrogen and captured carbon. At least that's how we do it. 

2.) If models implement it as a conversion from hydrogen and CO2 to liquids, then it is extra effort in the model reporting to split the liquids into those where the hydrogen has been produced from electricity and those where the hydrogen has come from other sources initially (biomass, gas etc.). The latter would then probably need to be reported under ``Secondary Energy|Liquids|Other``. I would find it confusing to interpret model data if outputs of the same technology get attributed to different variables just because the inputs were derived from different sources. I find it much more straightforward to see that a model produces a lot of liquids from hydrogen and then check its hydrogen mix for the sources instead of mixing these two levels. 

There are two counterarguments to this, which come to my mind. 

3.) One may argue that people are mostly interested in e-fuel production (that is, synthetic fuels from electrolytic hydrogen) and that this may be the dominant source of hydrogen in the scenarios. I'd say though, that one can easily calculate the e-fuel share in ``Secondary Energy|Liquids|Hydrogen`` by using the ``SE|Hydrogen...`` variables in any post-processing. As a modeler, I prefer having the hydrogen -> liquids conversion to check the model dynamics / results. Even in the real world, there may be Fischer-Tropsch plants that use captured carbon from some nearby CO2 source and hydrogen from the grid where the hydrogen of different origins may be fed in. That speaks in favor of considering this as a separate step instead of lumping everything together with the electrolysis step. 

4.) Using ``Secondary Energy|Liquids|Hydrogen`` may run into some inconsistencies as well if not properly defined. You can argue that if this should contain all liquid hydrocarbon fuels where some form of hydrogen was an intermediate product, this would basically include all kinds of synthetic fuels where synthesis gas (H2 + CO) is fed into a Fischer-Tropsch process. This would include Gas-to-Liquids or Biomass-to-Liquids, which so far I would see under ``SE|Liquids|Fossil`` and ``SE|Liquids|Biomass`` respectively. I think, this should be made clear in the definition, e.g. by calling it ``Secondary Energy|Liquids|Hydrogen w/ CCU``  so that it is clear that at some point there was a pure CO2 stream captured, which was combined with H2. In a real-world plant, this definition might seem a bit arbitrary as there might be never pure "H2" or "CO2" at any point and different gas streams may be combined. But, for the modeling I think it helps in interpreting the model results around hydrogen, e-fuels and synthetic fuels (understood broadly as including all PtL, BtL, GtL etc.). 

To sum up: I'd be in favor of replacing ``Secondary Energy|Liquids|Electricity`` by  ``Secondary Energy|Liquids|Hydrogen w/ CCU`` or a similar formulation. This would then also affect the ``Final Energy|Liquids|...`` variables. The same ofc for the ``Secondary Energy|Gases|...`` variables. 

Let me know what you think about this.  In particular, by which energy carrier mapping you model synthetic fuel / e-fuel production, so far. 

I did not find a tag for the energy group. Could we add one?

Tagging @pkyle and @christophbertram as they discussed a related issue #47 .

**Comment by pkyle** [comment]
This is a really good discussion question; I see several issues at stake:
1) Some technologies of secondary energy production, for liquid fuels in particular, involve synthesis of multiple sources (energy/feedstocks), each of which may be "primary" or "secondary" energy, which makes attribution to a single source impractical; and
2) The energy/feedstock sources of any synthesis technology may be produced on-site, effectively encapsulated within a model's production technology, or alternatively, a model's technology may explicitly take "intermediate" inputs. As simple examples that have always been in the models, and as such are already addressed in our existing reporting practices, wind electrolysis (i.e., wind -> electricity -> hydrogen) could or could not have the intermediate production of electricity reported. I believe in the past it has not been reported; i.e. the wind-electricity used for direct on-site electrolysis of hydrogen is not reported under `Secondary Energy|Electricity|Wind` or the associated parent categories. Similarly, biomass IGCC power plants implicitly have intermediate production and use of hydrogen, but this is reported as `Secondary Energy|Electricity|Biomass`, with no reference to hydrogen production and use.
Some of these "intermediate" transformations that used to be encapsulated within individual secondary energy production technologies (and not reported at all to the data templates) are starting to be broken out individually, and because the intermediate commodities themselves may be purchased from a pooled market, even they can't be simply assigned to a single energy source from which they were produced.
I don't see an easy way out of this, whereby we could continue to report all secondary energy commodities' production assigned to specific energy/feedstocks, in ways that are consistent and meaningful across models and across production technologies. Perhaps if we conducted a survey amongst models, we could figure out a set of technology categories that captures useful information and to which the output of each production technology in each model could be assigned, without requiring post-hoc disaggregation of individual technologies to separate reporting categories. I think as a starting point, the reporting categories in the templates ought to be less detailed than the technologies in the models.
In response to the suggested category of `Secondary Energy|Liquids|Hydrogen`, all liquid fuels technologies have a hydrogen input, even oil refining, irrespective of whether it's explicitly represented in the models; conversely, no liquid fuel production technologies take only hydrogen, as there would also need to be a carbon source. In the end it may be best to just leave the secondary energy production categories broad, and focus the analysis on the inputs to these commodities as a whole; not sure.
